src/models/teams.rs

```diff
@@ -20,6 +20,27 @@ pub struct Team {
     pub members: Vec<TeamMember>,
 }
 
+bitflags::bitflags! {
+    #[derive(Serialize, Deserialize)]
+    pub struct Permissions: u64 {
```

Comment:
It's not a requirement, but it could be useful to use `1 << n` notation for this as we add more permissions flags.